app/Jobs/PollForMediaWikiJobsJob.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Wiki;
+use Illuminate\Support\Facades\Http;
+
+class PollForMediaWikiJobsJob extends Job
+{
+    public function handle(): void
+    {
+        $wikis = Wiki::all()->pluck('domain');
+        foreach ( $wikis as $wikiDomain ) {
+            if ($this->hasPendingJobs( $wikiDomain )) {
+                $this->enqueueWiki( $wikiDomain );
+            }
+        }
+    }
+
+    private function hasPendingJobs( string $wikiDomain ): bool
+    {
+        $response = Http::withHeaders([
+            'host' => $wikiDomain
+        ])->get(
+            getenv('PLATFORM_MW_BACKEND_HOST').'/w/api.php?action=query&meta=siteinfo&siprop=statistics&format=json'
+        );
+
+        if ($response->failed()) {
+            $this->fail(
+                new \RuntimeException(
+                    'Failure polling wiki '.$wikiDomain.' for pending MediaWiki jobs: '.$response->clientError()
+                )
+            );
+
+            return false;
+        }
+
+        $pendingJobs = data_get($response->json(), 'query.statistics.jobs', 0);
+        return $pendingJobs > 0;
+    }
+
+    private function enqueueWiki ( string $wikiDomain ): void
+    {
+        dispatch(new ProcessMediaWikiJobsJob( $wikiDomain ));
```

Comment:
@tarrow now that we limit concurrency on Kubernetes level, do you think it still makes sense to fan out to another job here? I'm a bit torn.

Pros I can see:
- cleaner if one job fails, i.e. no need to try/catch here
- reduces execution of this job
- allows for paralellizing if ever needed

Cons:
- Harder to understand
- Performance problems are probably negligible right now, i.e this might just be overengineering